CMakeLists.txt

```diff
@@ -0,0 +1,184 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(opus LANGUAGES C)
+
+include(opus_config.cmake)
+include(opus_sources.cmake)
+include(opus_functions.cmake)
+
+add_library(opus ${opus_sources} ${opus_sources_float})
+
+set_target_properties(opus
+                      PROPERTIES SOVERSION
+                                 ${OPUS_LIBRARY_VERSION_MAJOR}
+                                 VERSION
+                                 ${OPUS_LIBRARY_VERSION})
+
+target_include_directories(opus
```

Comment:
Missing `$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>`.

Don't forget to move `include(GNUInstallDirs)` before this command.